src/controller.rs

```diff
@@ -88,9 +88,12 @@ impl<'b> Controller<'b> {
             clircle::Identifier::stdout()
         };
 
+        const BUF_W_SZ: usize = 1 << 14;
         let mut writer = match output_buffer {
             Some(buf) => OutputHandle::FmtWrite(buf),
-            None => OutputHandle::IoWrite(output_type.handle()?),
+            None => {
+                OutputHandle::IoWrite(BufWriter::with_capacity(BUF_W_SZ, output_type.handle()?))
```

Comment:
That might very well be the case. If so, it would be good to add a comment here explaining that. So that anyone that gets the same idea will find the comment when they are about to change the code.